priam/src/main/java/com/netflix/priam/notification/BackupNotificationMgr.java

```diff
@@ -51,6 +53,13 @@ public BackupNotificationMgr(
         this.notificationService = notificationService;
         this.instanceInfo = instanceInfo;
         this.instanceIdentity = instanceIdentity;
+        this.notifiedBackupFileTypesSet = new HashSet<>();
+        if (!StringUtils.isBlank(this.config.getBackupNotifyComponentIncludeList())) {
+            notifiedBackupFileTypesSet.addAll(
+                    Arrays.stream(this.config.getBackupNotifyComponentIncludeList().split(","))
+                            .map(AbstractBackupPath.BackupFileType::fromString)
```

Comment:
You will get RuntimeException here if value contains illegal value. What does the error handling looks like?  I would recommend to reject the configuration change and keep StringUtils.EMPTY if it has illegal value. Also, adding javadoc for failure behavior in the configuration woud be a good idea. 

Also, this value will be updated only at startup. Can you make this dynamic so we don't need to restart Priam if we wish to change the value.